pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -154,7 +164,7 @@ type CiliumBGPVirtualRouter struct {
 	// to the configured neighbors.
 	//
 	// +kubebuilder:validation:Optional
-	ExportPodCIDR bool `json:"exportPodCIDR"`
+	ExportPodCIDR *bool `json:"exportPodCIDR"`
```

Comment:
Is there going to be any backward compatibility issues?

Reply:
Ah read the description again - no concerns if these changes are backward compatible.

Reply:
`omitemty` fixed. As this PR aims to keep backward compatibility, renaming or changing the type other than pointer change is not possible. If there will be a need for that we can do it later as a separate PR with some migration logic.